src/index.js

```diff
@@ -291,10 +302,10 @@ export default class Headroom extends Component {
     if (this.state.animation) {
       innerStyle = {
         ...innerStyle,
-        WebkitTransition: 'all .2s ease-in-out',
-        MozTransition: 'all .2s ease-in-out',
-        OTransition: 'all .2s ease-in-out',
-        transition: 'all .2s ease-in-out',
+        WebkitTransition: `transform ${this.props.speed}ms ${this.props.easing}`,
```

Comment:
Jan, I'm testing this and the component doesn't disappear from view on scroll: I think it might be to do with these ... I'm testing some more but perhaps you could also take a look?

Reply:
Hey Ben, sure I'll have a look at it later today. Thanks ☺️

Reply:
@endymion1818 I just tested it and I cannot reproduce the issue, but maybe I'm missing something. The css styles are applied correctly to the inner div inside `headroom-wrapper`, when passing individual props to Headroom component. Could it be a browser specific issue? I've created a [codesandbox demo](https://codesandbox.io/s/jolly-forest-5xhsz). the code `react-headrom` is taken from the forked repo (master branch).